includes/Hooks.php

```diff
@@ -248,6 +251,54 @@ public static function onCodeEditorGetPageLanguage( $title, &$lang, $model, $for
 		return true;
 	}
 
+    /**
+     * @param $old
+     * @param $new
+     * @param $userIdentity
+     * @param int $pageid
+     * @param int $redirid
+     * @param string $reason
+     * @param $revision
+     * @return true
+     */
+    public static function onPageMoveCompleting($old, $new, $userIdentity, int $pageid, int $redirid, string $reason, $revision) {
```

Comment:
I'd only add a comment in the function doc describing what we are doing here and why, so it's easier for us during MW upgrades